src/app/services/tasksApi.ts

```diff
@@ -19,7 +19,18 @@ export const tasksApi = api.injectEndpoints({
             query: () => MINE_TASKS_URL,
             providesTags: ['Mine_Tasks'],
         }),
+        updateTasks: builder.mutation({
+            query: ({ id, ...patch }) => ({
+                url: `/tasks/${id}`,
+                method: 'PATCH',
+                body: patch,
+            }),
+        }),
```

Comment:
Nit: Hello @Pratiyushkumar...I seen @bharati-21 also adding some updateTasks RTK query in this file
Her pr code:- https://github.com/Real-Dev-Squad/website-status/pull/643/files#diff-4ac2244e7dd24a92566772fd14cbb639b24f63f52996f94de90d8f31107787c7R42

Reply:
will have to sync up with this with bharti

Reply:
have talked to Bharati, she will manage it.